src/main.rs

```diff
@@ -56,21 +56,17 @@ fn parse_arg_from_client_input(raw: &String) -> Result<String> {
 }
 
 fn parse_target_process_number_from_client_input(
-    raw: &String,
-    client_arg: &String,
+    raw: &String
 ) -> Result<Option<u32>> {
     if let Some(cmd) = raw.split_whitespace().skip(1).next() {
         if let Some(index) = cmd.find(":") {
```

Comment:
Can you use rfind to handle the case with a process name like `term:42` (`start term:42:1`) pls :)